Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -584,6 +633,10 @@ internal void ProcessNavigation(ref NavigationModel navigationState)
         // Hide this while we still have to figure out what to do with this.
         private float m_TrackedDeviceDragThresholdMultiplier = 2.0f;
 
+        [Tooltip("Transform representing the real world origin for tracking devices.")]
```

Comment:
It may be worth adding to this tooltip and xml-doc of the property to say this should typically be set to an object representing an "XR Rig". We talk about a rig in https://docs.unity3d.com/Manual/configuring-project-for-xr.html and nudging the user to set this to the expected Transform would probably help.